src/components/OSCALProfileCatalogInheritance.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { profileCatalogInheritanceData } from "../test-data/CommonData";
+import OSCALProfileCatalogInheritance from "./OSCALProfileCatalogInheritance";
+
+function profileCatalogInheritanceRenderer() {
+  render(
+    <OSCALProfileCatalogInheritance
+      inheritedProfilesAndCatalogs={profileCatalogInheritanceData}
+    />
+  );
+}
+
+test("OSCALProfileCatalogInheritance displays top-level inherited objects", async () => {
+  profileCatalogInheritanceRenderer();
+  const result = await waitFor(
+    () => screen.getByText("Catalog: Example Catalog"),
+    { timeout: 5000 }
+  );
+  expect(result).toBeVisible();
+});
+
+test("OSCALProfileCatalogInheritance displays nested inherited objects", async () => {
+  profileCatalogInheritanceRenderer();
+  const inheritanceButton = await screen.findByRole(
```

Comment:
It seems like it would make sense to at least test the expansion and collapse of the tree to ensure everything works and is displayed as expected in both cases.

Reply:
But this test can be rewritten once everything is expanded by default.

Reply:
OK, that's fine.